doc/akri_connector/connector-metadata-schema.json

```diff
@@ -34,7 +34,7 @@
         },
         "topic": {
           "type": "string",
-          "description": "The default MQTT topic that will be published to"
+          "description": "The default MQTT topic that will be published to. Values are allowed to contain deploy-time parameters like 'mqtt/{deviceName}/{inboundEndpointName}/myMessages'. The supported parameter tokens are: {deviceName}, {inboundEndpointName}, {assetName}, {connectorClientId}, {datasetName}, {eventName}, {streamName}, {managementGroupName}, {datasetDatapointName}, {eventDatapointName}, {namespace}, and {endpointType}."
```

Comment:
datapoints wouldn't have destinations, so I'm not sure there's a meaningful value that could replace those tokens. Also not sure if you want to call out that {datasetName} could only be used as a token for a dataset?

Reply:
Good point x 2